libcxx/include/__iterator/istream_iterator.h

```diff
@@ -58,6 +58,9 @@ class _LIBCPP_TEMPLATE_VIS istream_iterator
       __in_stream_ = nullptr;
   }
 
+  // LWG3600 Changed the wording of the copy constructor. In libc++ this constructor
+  // can still be trivial after this change.
+
```

Comment:
I'd much rather just mention that our copy construct or trivial as a conforming extension. I don't think it's useful in the future to know which LWG issue changed that.

Reply:
I added the LWG issue number to let the reader know we did look at this constructor for LWG3600.

Reply:
I'm not sure what you're trying to say with "we did look at this constructor for LWG3600". Do you mean we checked whether we're conforming because LWG3600 exists? If you mean that, I'm really not sure why anybody would care.

Reply:
I care a lot. In the past we had several LWG issues closed as `Completed` or `Nothing To Do` when that was not correct. By explicitly mentioning it, it will safe time when somebody notices a difference between libc++ and the Standard.